applications/tari_indexer/src/substate_manager.rs

```diff
@@ -260,35 +263,135 @@ impl SubstateManager {
         Ok(nfts)
     }
 
-    pub async fn get_event_from_db(
-        &self,
-        template_address: TemplateAddress,
-        tx_hash: PayloadId,
-    ) -> Result<Vec<EventData>, anyhow::Error> {
-        let mut tx = self.substate_store.create_read_tx()?;
-        let events = tx.get_events(template_address, tx_hash)?;
-        Ok(events)
-    }
-
     pub async fn save_event_to_db(
```

Comment:
Not an async function
```suggestion
    pub fn save_event_to_db(
```